pkg/controller/stacks/stack/stack.go

```diff
@@ -50,6 +55,12 @@ var (
 	log              = logging.Logger.WithName(controllerName)
 	resultRequeue    = reconcile.Result{Requeue: true}
 	requeueOnSuccess = reconcile.Result{RequeueAfter: requeueAfterOnSuccess}
+
+	roleVerbs = map[string][]string{
+		"admin": {"get", "list", "watch", "create", "delete", "deletecollection", "patch", "update"},
+		"edit":  {"get", "list", "watch", "create", "delete", "deletecollection", "patch", "update"},
```

Comment:
are admin and edit supposed to be the same here?

Reply:
We originally had an invalid `edit` role prescribed in the design doc. `update` and `patch` seem reasonable, but the Kubernetes `edit` role adopts create, delete, and delete collection as well. 

`admin` gets the advantage of working with secrets.

@lukeweber thoughts?

Reply:
The goal was to model based on upstream kubernetes verb behavior for their built in roles. 

https://kubernetes.io/docs/reference/access-authn-authz/rbac/

admin | None | Allows admin access, intended to be granted within a namespace using a RoleBinding. If used in a RoleBinding, allows read/write access to most resources in a namespace, including the ability to create roles and rolebindings within the namespace. It does not allow write access to resource quota or to the namespace itself.
-- | -- | --
edit | None | Allows read/write access to most objects in a namespace. It does not allow viewing or modifying roles or rolebindings.
view | None | Allows read-only access to see most objects in a namespace. It does not allow viewing roles or rolebindings. It does not allow viewing secrets, since those are escalating.

From this, admin is manage role bindings and all behavior of edit, edit can manage most objects except the bindings, and view can view most objects except secrets.

Reply:
Good point, @lukeweber.

I've added rolebinding and clusterrolebinding rules in 83e7fbf3